tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -594,4 +594,46 @@ public function userUnmountsShareUsingTheGraphApi(string $user):void {
 		$this->featureContext->setResponse($this->unmountShare($user, $itemId, $shareSpaceId));
 		$this->featureContext->pushToLastStatusCodesArrays();
 	}
+
+	/**
+	 *
+	 * @param string $user
+	 * @param string $resourceId
+	 * @param string $shareSpaceId
+	 *
+	 * @return ResponseInterface
+	 */
+	public function mountShare(string $user, string $resourceId, string $shareSpaceId):ResponseInterface {
```

Comment:
```suggestion

```

Reply:
mountShare will be helper function while other step definition might call this function . So, both given and when statement can reused this function rather than linking function of step defintion

Reply:
@S-Panta Are there any other step-defs where it is used?

Reply:
Let's keep the function. We will need it very soon for https://github.com/owncloud/ocis/issues/8717

Reply:
@S-Panta YAGNI

Reply:
There shall be more tests added https://github.com/owncloud/ocis/issues/8486 as soon as this PR gets merged. `mount` and `unmount` will be given steps for the scenario that will be added. This isn't YAGNI IMO.

Reply:
IMO creating a new function to call another function does not make any sense.

Reply:
Refactored code. No YAGNI now ig.